packages/protocol/contracts/thirdparty/LibFixedPointMath.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
not to be nitpicky but, i believe UNLICENSED is an issue. i think this breaks our ability to be MIT licensed technically, even though I doubt it will be a real issue with the maintainer. i reached out to recmo on twitter to see if we can use the MIT license.

Reply:
Can we create an issue and fix it later?